sdcm/mgmt/operator.py

```diff
@@ -87,6 +87,7 @@ class ScyllaOperatorRepairTask(ScyllaOperatorTaskBaseClass):
     # small_table_threshold enable small table optimization for tables of size lower than given threshold.
     # Supported units [B, MiB, GiB, TiB] (default "1GiB").
     small_table_threshold: str = None
+    ignore_down_hosts: bool = None
```

Comment:
Is this in ScyllaOperator?

Reply:
The link above is the latest dev version of the scylla-operator.
And the `ignore_down_hosts` is absent there.

It means, a bug must be filed against scylla-operator to add support for this param and then use in here in a comment describing that it's usage must be enabled only when a new scylla-operator release comes out with this bug fixed.

Reply:
@zimnx it was just never added, it's an easy fix to add it though